ai4good/webapp/run_model_page.py

```diff
@@ -50,7 +50,7 @@ def profile_selector():
     return dbc.Row([
         dbc.Col(
             html.Div([
-                html.Label('<a href="https://raw.githubusercontent.com/AIforGoodSimulator/model-server/master/fs/params/Parameters_mouse-over_des.csv">Profile</a>', style={'font-weight': 'bold'}),
+                html.Label([html.A('Profile', href='https://github.com/AIforGoodSimulator/model-server/blob/master/fs/params/Parameters_mouse-over_des.csv', target="_blank")], style={'font-weight':'bold'}),
```

Comment:
why do we have absolute link here?

Reply:
Hi @titorenko because this is a quick hack, do you suggest:
- load from local
- put it in an table or separate page?
- the page is for data scientists or anyone who would like to view a full list of parameters from all 3 models

Reply:
Hi @titorenko the above is now fixed (by others in the dev). Thanks